history-of-the-quakers/original/04-book-1-chapter-1.adoc

```diff
@@ -1,25 +1,30 @@
 == CHAPTER 1
 
-Birth of George Fox.--His early turn to Religion.--Is put apprentice.--Religious
-Thoughtfulness predominant in his Mind,
-and its Effects.--He leaves his Relations.--Falls under
-great Trouble of mind.--Applies to several Priests,
-but receives no Relief.--Withdraws from the public Worship.--Passeth
-his Time in Retirement.--Grows in religious Experience.
+[.chapter-synopsis]
+* Birth of George Fox
+* His early turn to Religion
+* Is put apprentice
+* Religious Thoughtfulness predominant in his Mind, and its Effects
+* He leaves his Relations
+* Falls under great Trouble of mind
+* Applies to several Priests, but receives no Relief
+* Withdraws from the public Worship
+* Passeth his Time in Retirement
+* Grows in religious Experience.
 
 George Fox was a native of Leicestershire, being born at Drayton in the Clay,
 in the said county, in the year 1624,
 of parents not considerable for their rank in the world,
 but Birth of respectable in their neighbourhood for piety, and integrity of life,
 who gave him a sober, although not a learned, education,
-in the way and worship of the national church: His mother especially,^
-footnote:[Penn`'s Preface to George Fox`'s Journal.]
+in the way and worship of the national church: His mother especially,
 a woman of qualifications superiour to the generality of her circumstances in life,
 observing the bent of his disposition to gravity and seriousness from his childhood,
 treated him with particular tenderness and indulgence,
 being careful not to discourage his virtuous inclinations,
 feeling much satisfaction in these early indications of
-a pious and religious turn of mind in her son.
+a pious and religious turn of mind in her son.^
+footnote:[Penn`'s Preface to George Fox`'s Journal.]
```

Comment:
Ok, so there's going to be a lot of these, but I think we also want you to start trying to notate the titles of books using the `[.book-title]#TITLE#` syntax, see here: https://flp-styleguide.netlify.com/#styling

The main trick with these is that the author often doesn't actually refer to the _full and complete book title_ -- rather often uses a shorthand, so you have to try to pick out what part is most "title-like" and wrap that in the book-title syntax. So for this, I would probably just grab the word "Journal", making it like this:

```ASCIIDOC
footnote:[Penn`'s Preface to George Fox`'s [.book-title]#Journal.#]
```

Reply:
I think there are 3 more references to Fox's journal in footnotes in this chapter as well, so let's get all those while we're at it.  When he references "Penn's Preface" -- I think we can leave those as is, because he's really referencing a _chapter_ of a book, and I don't think `[.book-title]` is appropriate there.